crypto/ocsp/ocsp_test.cc

```diff
@@ -218,3 +525,101 @@ TEST(OCSPTest, TestBasic) {
   basic_response = bssl::UniquePtr<OCSP_BASICRESP>(OCSP_response_get1_basic(ocsp_response.get()));
   ASSERT_TRUE(basic_response);
 }
+
+TEST(OCSPTest, TestGoodOCSP) {
```

Comment:
You have some shared logic between the two good tests you have now. I envision more good tests covering more accept states. Could consider abstracting these tests further to make that easier.

Reply:
You know the expected status code for each test case. Pass that as part of the test case parameters.

Should be abstracted when adding more tests.

Reply:
Abstraction added :)

Reply:
Nice nice :)